website/static/files/build_orca_inventory_refined.sql

```diff
@@ -123,6 +126,8 @@ BEGIN
       IS 'Cumulus S3 bucket where the file is thought to be stored.';
     COMMENT ON COLUMN files.key_path
       IS 'Full AWS key path including file name of the file (does not include bucket name) where the file resides in ORCA.';
+    COMMENT ON COLUMN files.multipart_chunksize_mb
```

Comment:
Looks like this column doesn't exist. It does, so @bhazuka, should I add the column and comment into develop?

Reply:
Normally I would say yes but, since we can hopefully get this temp branch merged into develop soon, I think it is probably ok to do it here.